packages/esm-patient-forms-app/src/index.ts

```diff
@@ -38,6 +38,12 @@ function setupOpenMRS() {
   defineConfigSchema(moduleName, {});
 
   return {
+    pages: [
+      {
+        load: getAsyncLifecycle(() => import('./forms/forms-see-all'), options),
+        route: 'patient/forms',
```

Comment:
Miss, e.g., the patient UUID etc.

According to the hyperlink we have something like `patient/forms/${patientUuid}/forms-see-all`, but here I just see `patient/forms`. Is the `forms-see-all` not required? The UUID should be. But the URL format indicates that the segment for patient UUID is actually the form UUID.

Why not have

```
/patient/${patientUuid}/forms
```

Reply:
I thought even if a part (begining part) of the string specified with route matches the URL it will load the page. If it is possible to assign a value like `route: 'patient/${patientUuid}/forms'` , I will do that.

Reply:
At Amazon you are also not going to a page "see-all-products" but "products". I am not sure why you don't follow general principles (i.e., things that you see around in the world, where best practices are already followed), but come up with things like that.

The routing mechanism can also use a regular expression. Have a look at the patient chart microfrontend to see how that can be done.

Reply:
Done. Thanks!

Reply:
I have changed the path format.